src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let calculateMen = people.filter(person => person.sex === 'm');
+
+  calculateMen = century
+
+    ? calculateMen.filter(men => Math.ceil(men.died / 100) === century
+    ) : calculateMen;
```

Comment:
Check here should be more specific because century theoretically can have valid falsy value - 0;

```suggestion
  calculateMen = century !== undefined
    ? calculateMen.filter(men => Math.ceil(men.died / 100) === century) 
    : calculateMen;
```